docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.4'
```

Comment:
@trungthanhtong - can we still run this via docker-compose for local development?

Reply:
@zlove We can still run it via docker-compose